project/src/pages/welcome-page/welcome-page.tsx

```diff
@@ -116,11 +107,15 @@ function WelcomePage({offerCount, offers}: WelcomePageProps): JSX.Element {
                 </ul>
               </form>
 
-              <OffersList offers={offers} offerCount={offerCount}/>
+              <OffersList offers={offers} offerCount={offerCount} onListOfferHover={onListOfferHover}/>
 
```

Comment:
есть критерий про именование обработчиков, проп пусть и дальше называется onListOfferHover, а вот сам обработчик назови через handle
```suggestion
              <OffersList offers={offers} offerCount={offerCount} onListOfferHover={handleListOfferHover}/>

```